src/System.Security.Cryptography.Primitives/src/System/Security/Cryptography/AsymmetricAlgorithm.cs

```diff
@@ -11,15 +11,11 @@ public abstract class AsymmetricAlgorithm : IDisposable
 
         protected AsymmetricAlgorithm() { }
 
-        public static AsymmetricAlgorithm Create()
-        {
-            return Create("System.Security.Cryptography.AsymmetricAlgorithm");
-        }
+        public static AsymmetricAlgorithm Create() =>
```

Comment:
Is it possible to add `[Obsolete]` (as error) to the reference assemblies so that devs targeting netstandard or netcoreapp will see the failure at compilation time?

Reply:
Last I knew: no (against our rules).  But we were/did/will build(ing) an analyzer pack of methods that are cross-platform risks and/or just generally disrecommended.  Rfc2898DeriveBytes overloads which don't take the HashAlgorithmName are on it, as are all of these.